opi/plugins/packman.py

```diff
@@ -31,3 +31,12 @@ def run(cls, query):
 			'gstreamer-plugins-bad',
 			'gstreamer-plugins-ugly',
 		])
+
+		if not opi.ask_yes_or_no("Do you want to install openh264 codecs from openh264 repository?", 'y'):
+			return
+		opi.add_openh264_repo(dup=True)
+
+		opi.install_packages([
+			'gstreamer-1.20-plugin-openh264',
```

Comment:
```suggestion
			'libgstopenh264.so()(64bit)',
```

How often does the gstreamer version and therefore this package name change?

Would it make sense to use some of this provides instead or add a version agnostic provide to the package spec?

```
$ rpm -q --provides gstreamer-1.20-plugin-openh264-1.20.3-1.suse1599.1.x86_64.rpm
gstreamer-1.20-plugin-openh264 = 1.20.3-1.suse1599.1
gstreamer-1.20-plugin-openh264(x86-64) = 1.20.3-1.suse1599.1
gstreamer1(decoder-video/x-h264)()(64bit)
gstreamer1(element-openh264dec)()(64bit)
gstreamer1(element-openh264enc)()(64bit)
gstreamer1(encoder-video/x-h264)()(64bit)
libgstopenh264.so()(64bit)
```

Reply:
this option will not work on aarch64 and other non-amd64 architectures. I'll think about how to do it differently.

Reply:
although, as I found out, it will work on aarch64, but the rest are probably not found now, so perhaps let it be as you suggest.